### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by keidarcy**
Greate repo to explain!
When I try to run your example, if my understanding is correct, I found something that should be fixed like this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,14 +17,14 @@ Node doesn't allow using `.js` extension for both `esm` and `commonjs` files in
 
 When running this package, you will get an error, depending on it's `type` field:
 
-- run `node cjs/test-esm.js` to get `require() of ES Module` error (if package has no `type` or `"type": "commonjs"`)
+- run `node cjs/test-esm.js` to get `require() of ES modules is not supported.` error (if package has no `type` or `"type": "commonjs"`)
 
-  - which means you can only use `import` syntax with this module in Node
-  - when running `node cjs/test-cjs.mjs`, you will not get an error
+  - which means you can only use `require` syntax with this module in Node
```
Comment (sheremet-va) [inline README.md]: How can this be true, if in this example we are requiring module and getting the error? 

Your fix implies it won't throw

File: README.md [hunk 2]
```diff
@@ -17,14 +17,14 @@ Node doesn't allow using `.js` extension for both `esm` and `commonjs` files in
...
+  - when running `node cjs/test-cjs.js`, you will not get an error
 
-- run `node esm/test-cjs.mjs` to get `The requested module is a CommonJS module` error (if package has `"type": "module"`)
+- run `node esm/test-cjs.mjs` to get `The requested module 'test-cjs' is a CommonJS module` error (if package has `"type": "module"`)
 
-  - which means you can only use `require` syntax with this module in Node
+  - which means you can only use `import` syntax with this module in Node
```
Comment (sheremet-va) [inline README.md]: Same here, example uses import and throws, which means you can only require it

**Review by sheremet-va — CHANGES_REQUESTED** [review]

**Comment by sheremet-va** [comment]
I have edited examples base on your feedback. What do you think now?

File: README.md (hunk 1 above)
Comment (keidarcy) [reply README.md]: Okay, there is some midunderstanding.

> run `node cjs/test-esm.js` to get `require() of ES Module` error (if package has no `type` or `"type": "commonjs"`)

In the module of "run node `cjs/test-esm.js`", package.json has `"type": "module"` => can only use `import`.

In my previous understanding, in the module of "if package has no `type` or `"type": "commonjs"`" => can only use `require`.

**Comment by keidarcy** [comment]
@sheremet-va 
Thanks, I'll close this PR.

**Pull request closed without merging**